kuma/static/js/wiki-compat-signal.js

```diff
@@ -2,6 +2,669 @@
     'use strict';
 
     var bcTable;
+    var bcSignalBlock;
+    var bcSignalCompleteBlock;
+    var sendReportButton;
+    var nextStepButton;
+    var stepsInfoSpan;
+    var bcSignalStep;
+    var bcSignalSteps = 2;
+
+    /**
+     * Toggles main block visibility and resets form completion progress
+     */
+    function toggleBcSignalBlock () {
+        if (bcSignalCompleteBlock.classList.contains('open')) {
+            bcSignalCompleteBlock.classList.remove('open');
+        } else {
+            bcSignalBlock.classList.toggle('open');
+        }
```

Comment:
I believe one can simply do:

```
bcSignalBlock.classList.toggle('open');
```

This will add or remove the class based on the presence(or lack thereof) of the class.